src/net/sourceforge/kolmafia/persistence/ConsumablesDatabase.java

```diff
@@ -873,30 +879,41 @@ public static final double getAdventureRange(final String name) {
       return 0.0;
     }
 
-    String cname = StringUtilities.getCanonicalName(name);
     boolean perUnit = Preferences.getBoolean("showGainsPerUnit");
     Double range = null;
 
+    var adventuresBoosted = areAdventuresBoosted(name);
+
     if (ConsumablesDatabase.getRawFullness(name) != null) {
-      int itemId = ItemDatabase.getItemId(name, 1, false);
-      boolean sushi = ConcoctionDatabase.getMixingMethod(itemId, name) == CraftingType.SUSHI;
-      boolean milk = KoLConstants.activeEffects.contains(ConsumablesDatabase.MILK);
+      boolean milk = Preferences.getBoolean("milkOfMagnesiumActive");
       boolean lunch =
           KoLConstants.activeEffects.contains(ConsumablesDatabase.GLORIOUS_LUNCH)
               || ConsumablesDatabase.BARREL_OF_LAUGHS.getCount(KoLConstants.activeEffects) >= 5;
       boolean gourmand = KoLCharacter.hasSkill("Gourmand") || KoLCharacter.hasSkill("Neurogourmet");
       boolean munchies = Preferences.getInteger("munchiesPillsUsed") > 0;
       range =
           ConsumablesDatabase.getAdventureMap(
-                  perUnit, !sushi && milk, !sushi && lunch, !sushi && gourmand, !sushi && munchies)
+                  perUnit,
+                  adventuresBoosted && lunch,
+                  adventuresBoosted && gourmand,
+                  adventuresBoosted && munchies,
+                  false)
```

Comment:
```suggestion
                  perUnit,
                  false,
                  adventuresBoosted && lunch,
                  adventuresBoosted && gourmand,
                  adventuresBoosted && munchies)
```

By comment text in `ConsumablesDatabase.calculateAdventureRange`, `gainEffect1` is ode or milk, `gainEffect4` is munchies pill.

I think the way you have it will lead to the wrong adventures expected for munchies pill.

Reply:
Ah ok thank you. I misunderstood the function signature. This all needs refactoring anyway